### DISCUSSION TITLE
Melcloud available for Homey w. bridge?

### DISCUSSION THREAD

**Discussion by bkange** [OlivierZal/com.melcloud#8] [Q&A] [answered]
@OlivierZal hi, is there any plans for the melcloud app to be available for Homey with only the bridge?

**Comment by OlivierZal** [comment, marked as answer]
Hi, unfortunately no.

To develop apps to be run on Homey Cloud, you need to belong to the company brand developing the official app, or be commissioned by it – what is not my case.

You’ll find details here: https://apps.developer.homey.app/guides/homey-cloud

Sorry…
Regards

**Reply by bkange** [reply]
Ok understood, thanks for quick answer.
